pages/linux/e2label.md

```diff
@@ -0,0 +1,7 @@
+# e2label
+
+> Change the label on an ext2/ext3/ext4 filesystem.
+
+- Change the volume label on a specific partition:
+
+`e2label {{/dev/sda1}} {{"my-label"}}`
```

Comment:
What do you think about changing it my-label to label_name ? That matches with the other token names that we have.

Reply:
Done.